public/css/search.css

```diff
@@ -0,0 +1,295 @@
+/* All */
```

Comment:
please remove your `css` and `js` folder inside `public` directory and please add this to your `.gitignore`
```
/public/css
/public/js
```